src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -146,8 +149,13 @@ public async Task SetInitialOrder(string contentType)
             }
         }
 
-        public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, bool publishedOnly)
+        public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, ContainedItemOptions containedItemOptions)
         {
+            if (containedItemOptions == null)
+            {
+                containedItemOptions = new ContainedItemOptions();
```

Comment:
throw a null ref exception if it's required.

Reply:
It was created this way, because the instance includes the status property.

Reply:
Throw a null reference exception if the value is required.